eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -31,7 +34,9 @@ run_e2e_test() {
         --env=CLOUDSERVER_ENDPOINT=${CLOUDSERVER_ENDPOINT} \
         --env=ZENKO_ACCESS_KEY=${ZENKO_ACCESS_KEY} \
         --env=ZENKO_SECRET_KEY=${ZENKO_SECRET_KEY} \
-        --env=ZENKO_SESSION_TOKEN=${ZENKO_SESSION_TOKEN} \
```

Comment:
why is the session token removed?

Reply:
changed back: [`feb5120` (#1207)](https://github.com/scality/Zenko/pull/1207/commits/feb51204e245798cd20df79a022fa5321d94ebd9)